src/webview.ts

```diff
@@ -1,13 +1,16 @@
 import * as vscode from "vscode";
 import validateColor from "validate-color";
 import { imagePreviewProviderViewType } from "./const";
+import * as path from 'path';
```

Comment:
```suggestion
import * as path from "path";
```

Small thing for consistency with the other imports

Reply:
oh, sorry.